doc/api/crypto.md

```diff
@@ -698,6 +698,33 @@ module):
 * `DH_UNABLE_TO_CHECK_GENERATOR`
 * `DH_NOT_SUITABLE_GENERATOR`
 
+## Class: DiffieHellmanGroup
+<!-- YAML
+added: v0.7.5
+-->
+
+Usage:
```

Comment:
OK, we're getting really close. Thanks for your patience with this. This doesn't follow the format of similar entries in the documentation. Put the text describing the class first, and the sample code after it. No need for lines like this one that say "Usage:" or "Example:".